org.newerth.savagexr.json

```diff
@@ -0,0 +1,121 @@
+{
+	"app-id": "org.newerth.savagexr",
+	"branch": "master",
```

Comment:
Wrong branch, either set it to stable or remove completely.

Reply:
removed.